src/modules/user/user.usecase.ts

```diff
@@ -518,4 +558,36 @@ export class UserUseCases {
       password,
     });
   }
+
+  async updatePassword(
+    user: User,
+    updatePasswordDto: UpdatePasswordDto,
+  ): Promise<void> {
+    const { currentPassword, newPassword, newSalt, mnemonic, privateKey } =
+      updatePasswordDto;
+
+    if (user.password.toString() !== currentPassword) {
+      throw new InvalidPasswordError();
+    }
+    await this.userRepository.updateById(user.id, {
+      password: newPassword,
+      hKey: new Buffer(newSalt),
```

Comment:
Is the new Buffer usage on drive-server update password functionality? `new Buffer` is currently [deprecated](https://nodejs.org/api/buffer.html#new-bufferarray). Use `Buffer.from(data, 'format')` instead